src/trupti/String/Assi25_26/MaximumDigits.java

```diff
@@ -0,0 +1,35 @@
+package trupti.String.Assi25_26;
+
+public class MaximumDigits {
+	String getMaximumDigits(String[] arr){
+		char c;
+		String maxDigitString="";
+		int maxDigitCount=0 , count=0;
+		
+		for(int outIndex=0;outIndex<arr.length;outIndex++) {
+			String str = arr[outIndex];
```

Comment:
count should be init with 0 for new word right, your logic break for below input
String [] arr={"P1u2r2va", "1K2omal", "4M4a5d5h1v1i93234","9S4hivan4i4", "1K2r3ati", "4M4a5d5h1v1i93"};